pywbem/__init__.py

```diff
@@ -45,6 +45,7 @@
 from ._recorder import *  # noqa: F403,F401
 from .config import *  # noqa: F403,F401
 from ._statistics import *  # noqa: F403,F401
+from ._logging import *  # noqa: F403,F401
```

Comment:
I was holding the change history change for the last of the parts of this change rather than piece parting it in.